src/main/cluster.ts

```diff
@@ -169,6 +169,12 @@ export class Cluster implements ClusterModel, ClusterState {
    * @observable
    */
   @observable isAdmin = false;
+  /**
+   * Is cluster marked as dead, for example due the invalid kubeconfig
+   *
+   * @observable
+   */
+  @observable isDead = false;
```

Comment:
And as I mentioned in my previous PR, I really think that adding this field is conflating what a cluster is.